src/assets/images/icons/check.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If this is a checkmark then we already have a `tick` icon.

Reply:
you are right, totally forgot. 
Just noticed, that the new check icon seems to be thinner than the existing one. Or maybe it's just the same behaviour as above 🤔